pkg/NETStandard.Library.NETFramework/NETStandard.Library.NETFramework.pkgproj

```diff
@@ -143,5 +147,13 @@
         <TargetPath Condition="'%(LibFile.TargetPath)' == '' ">build/%(LibFile.TargetFramework)/lib%(LibFile.SubFolder)</TargetPath>
       </File>
     </ItemGroup>
+
+    <ItemGroup>
```

Comment:
I didn't do that because this is specific to this one project and it doesn't set IncludeSymbolsInPackage.